Node/src/botbuilder-teams.d.ts

```diff
@@ -1036,6 +1044,18 @@ export class TeamsChatConnector extends builder.ChatConnector {
   *  @param handler The function to execute when an invoke request to submit task module results is received.
   */
   public onTaskModuleSubmit(handler: TaskModuleSubmitHandlerType): void;
+
+  /**
+  *  Set a handler that is called when an invoke request from command actions to fetch task is received .
```

Comment:
Hard to understand. Can you rephrase it to:
 *  Set a handler for compose extension invoke request made when <describe situation here>

Reply:
Same for onComposeExtensionFetchTask

Reply:
fixed